classes/order/OrderState.php

```diff
@@ -162,4 +162,21 @@ public function isRemovable()
     {
         return !($this->unremovable);
     }
+
+    /**
+     * @param string $name
+     * @param int $idLang
+     * @param int|null $idOrderState
+     *
+     * @return int
+     */
+    public static function countByLocalizedName(string $name, int $idLang, ?int $idOrderState): int
```

Comment:
```suggestion
    public static function countByLocalizedName(string $name, int $idLang, ?int $excludeIdOrderState): int
```
Suggestion (not blocking): change the variable name to highlight its meaning, this identifier is used to _exclude_ results not to include (and explain this in phpDoc)

Reply:
Agreed, either that or you could remove this condition and check that `$count == 0` in your validator And maybe this method would have more use cases without this extra condition

Reply:
Done @matks